INSTALL

```diff
@@ -70,7 +70,7 @@ The recipes for these can be found at https://github.com/conda-forge/feedstocks.
     Python package for the manipulation and analysis of planar geometric
     objects.
 
-**pyshp** 1.1.4 or later (https://pypi.python.org/pypi/pyshp)
+**pyshp** 2.0 or later (https://pypi.python.org/pypi/pyshp)
```

Comment:
Not available in most distros, so I'm not sure about bumping the requirement just yet.

Reply:
But then Cartopy can't be updated, as a package. I think that's a bit of a high price just to get a `close` method. Updating pyshp 1 to 2 may not happen as it include API breaks (assuming it follows semver), whereas Cartopy 0.18 may not.

Reply:
So you're basically saying that distros won't package pyshp 2 because it could be an API breaking change, whereas they would package cartopy because it is a minor change?

I accept this argument, and am happy to have undefined behaviour for pyshp 1 when calling the close method (an Exception for sure).

Reply:
Well, I can't say it's true for every distro, but at least Fedora tries to avoid API breaks in existing releases. Plus, I'm not the owner of the pyshp package, so wouldn't really be able to bump it myself, while I can do Cartopy at least. I have opened a bug report about the new pyshp, so hopefully the maintainer can take a look into updating it.

Reply:
I've updated the PR to reflect this conversation in our requirements files. I didn't update the INSTALL file but am happy enough to do so if that's important (it wasn't to me as it is a non-functional document that is only there to give a hint of what is required - the requirements/ files are the canonical source of what is necessary).